README.md

```diff
@@ -15,7 +15,7 @@ Purpose
 Demo
 ====
 
-  You can try an online demo here: http://code.alaiwan.org/dscripten/full.html
+  You can try an online demo here: http://code.alaiwan.org/dscripten/game.html
```

Comment:
Thanks, but it seems the old URL is correct.

Reply:
Oh, sorry. That is my mistake.